docs/docs/task-handling.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Task Handling
+---
+
+Marathon handles tasks in the following ways. 
+
+Can a user access these messages? How? Via the API? The DC/OS CLI?
+
+# Terminal states
+
+```
+case TASK_ERROR => Error
+```
+The task description contains an error.
+
+```
+case TASK_FAILED => Failed
+```
+The task failed to finish successfully. After Marathon marks the task as failed, it expunges the task and starts a new one.
+
+```
+case TASK_DROPPED => Dropped
+```
+The task failed to launch because of a transient error. The task's executor never started running. Unlike TASK_ERROR, the task description is valid -- attempting to launch the task again may be successful.
+
+```
+case TASK_GONE => Gone
+```
+
+The task was running on an agent that has been shutdown (e.g., the agent become partitioned, rebooted, and then reconnected to the master; any tasks running before the reboot will transition from UNREACHABLE to GONE). The task is no longer running. After Marathon marks the task as gone, it expunges the task and starts a new one.
+
+```
+case TASK_GONE_BY_OPERATOR => Gone
+```
+The task was running on an agent that the master cannot contact; the operator has asserted that the agent has been shutdown, but this has not been directly confirmed by the master. If the operator is correct, the task is not running and this is a terminal state; if the operator is mistaken, the task might still be running, and might return to the RUNNING state in the future. After Marathon marks the task as failed, it expunges the task and starts a new one.    
+
+```
+case TASK_FINISHED => Finished
+```
+The task finished successfully.
+
+```
+case TASK_UNKNOWN => Unknown
+```
+The mster has no knowledge of the task. This is typically because either (a) the master never had knowledge of the task, or (b) the master forgot about the task because it garbaged collected its metadata about the task. The task may or may not still be running. When Marathon receives the Unknown message, it expunges the task.
+
```

Comment:
Yes. Marathon tries to keep the number of active tasks or rather instances constant. If a task is unknown and thus expunged it is not counted as active.